docker/tag

```diff
@@ -1 +1 @@
-divide/conjur-dotnet-build:20180102T200617Z
+conjurinc/dotnet-build:20160308T103109Z
```

Comment:
This tag seems like it needs an update or maybe even complete removal

Reply:
this actually is really weird, and seems worth fixing.

I'm not sure what it's used for, so maybe it can just be deleted. but other projects build the docker tag from the VERSION file, and then do something like [this](https://github.com/cyberark/secretless-broker/blob/master/bin/build_utils#L20)

Reply:
It looks like `docker/Makefile` creates this and then it gets used as the tag when pushing to dockerhub. The `REPO`variable  is off too. I think I should make an issue to redo the whole dockerhub tagging and pushing process to model the secretless one

Reply:
You can just let this be part of #50